### DISCUSSION TITLE
How to disable Bevy application from generating .log binary files in the directory?

### DISCUSSION THREAD

**Discussion by imsfc** [bevyengine/bevy#11668] [Q&A] [answered]
```rs
fn main() {
    App::new()
        .add_plugins(DefaultPlugins)
        .run();
}
```

This is the result of building and running with `cargo build --release`.
Every time the program runs, it generates a new .log file.

![PixPin_2024-02-03_02-23-27](https://github.com/bevyengine/bevy/assets/30293082/15858e0a-2a3c-4c40-9833-7ebc2d20a710)

This is the content of the binary file.

![PixPin_2024-02-03_02-28-12](https://github.com/bevyengine/bevy/assets/30293082/c1b1bb36-ee97-478a-a4bd-fae540a734af)

When using MinimalPlugins, .log files are not generated. It seems that .log files are generated by RenderPlugin, but I'm not certain.

This is my system information: SystemInfo { os: "Windows 11 Pro", kernel: "22631", cpu: "13th Gen Intel(R) Core(TM) i9-13900KF", core_count: "24", memory: "63.8 GiB" }

**Comment by afonsolage** [comment]
I don't think this is related to Bevy, since Bevy is unable to log to file at moment. See https://github.com/bevyengine/bevy/pull/5342.

I'm also using Windows 11 and never got this log. This maybe be generated by the IDE you are using (if Visual Studio) or you have some build system which does this.

![image](https://github.com/bevyengine/bevy/assets/1176452/dd0f4178-b87b-444f-92e0-708134069e6f)

**Reply by imsfc** [reply]
Thank you for the response, but I have tried both the MSVC and GNU toolchains under Windows, and they still generate a .log file regardless of whether it's in dev or release mode. I am quite certain that it is produced by RenderPlugin, but my proficiency in Rust is limited, and I am unable to read the source code of RenderPlugin.

**Comment by imsfc** [comment]
Latest discovery: Running in Windows Sandbox does not produce a .log file. Additionally, running binaries built by others on my physical machine does generate a .log file. Therefore, it can be concluded that it is related to my hardware (as the Windows system has been freshly reinstalled). Another confirmed point is that the .log file is generated by the Bevy RenderPlugin, but only under specific circumstances. So, what could be the reason for this?

(This image displays the test results of a Bevy game binary compiled by someone else, running on my computer, sourced from: [https://github.com/Dreamtowards/Ethertum](https://github.com/Dreamtowards/Ethertum))
![PixPin_2024-02-04_02-16-35](https://github.com/bevyengine/bevy/assets/30293082/3ec0b0f8-5d74-4c4d-b8fd-632d2d9db6f7)

**Comment by imsfc** [comment]
The .log file is unexpectedly in binary format, so I am unable to extract information from it. (I am frustrated.)

I have uploaded the file. Is there anyone who can understand it? [ethertia.20240204-015603-320.log](https://github.com/bevyengine/bevy/files/14152484/ethertia.20240204-015603-320.log)

**Reply by oh-wind** [reply]
Did you solve this? I have the same problem

**Comment by oh-wind** [comment, marked as answer]
Uninstall WeGame and delete all its files can solve it.

Edit: The reason seems to be because of a fake verification layer provided by WeGame, which is in WeGame's `CrossVulkanLayer64.dll` and Vulkan loads it when your program starts.

**Reply by oh-wind** [reply]
FUCK TENCENT
